apps/prairielearn/elements/pl-number-input/pl-number-input.mustache

```diff
@@ -96,20 +125,21 @@
         });
     });
 </script>
+{{#show_correct}}
```

Comment:
Slight regression, this was changed at some point to do this through the grading logic function and not a separate key. See discussion: https://github.com/PrairieLearn/PrairieLearn/pull/7934#discussion_r1253733443

Reply:
By this, you mean that if the student answer submission is correct, then the answer used for grading should always be shown?

I definitely remember we have talked about this earlier at some point, I dont know if it's the merging nightmare that introduced this regression while fixing the [regression from earlier](https://github.com/PrairieLearn/PrairieLearn/pull/7934#discussion_r1285124690)

Reply:
> By this, you mean that if the student answer submission is correct, then the answer used for grading should always be shown?

Yes

> I definitely remember we have talked about this earlier at some point, I dont know if it's the merging nightmare that introduced this regression while fixing the [regression from earlier](https://github.com/PrairieLearn/PrairieLearn/pull/7934#discussion_r1285124690)

Yeah, I think this got lost somewhere. I think one of the earlier versions (pre-merge) had this as part of the logic in the grading function, so it shouldn't be _too_ hard to fish out that code.

Reply:
I had a quick look at the mustache file from the earlier commits before the big merge, and it looks like the code snippet (the div showing the correct answer used for grading) is always nested between `show_correct` for some reason. But since the line of code is nested between `{{#correct}}`, I think we could just take out the `show_correct` altogether since everything nested between `correct` wont be shown unless it's a correct 100% full credit submission (I think).

Reply:
I agree, I think that everything in `{{#show_correct}}` can be removed and handled more cleanly by returning it as feedback in the `grade()` function, where it will get displayed automatically.

Reply:
RIGHT, now memories are coming back. I think what we did was append the correct answer used for grading inside whatever feedback the student is also getting. Now I understand why the code snippet in question seems like an anachronism. 

So this is what people mean by when you code, you are both the perpetrator and the detective.

Reply:
Note to self on a discussion we probably had: I think we also talked about not liking the "this is the correct answer used for grading" being hardcoded in mustache (since this is ugly), and instead everything should be done in the `grade()` function. You'll probably have to dig up earlier reviews to find the traces of evidence of this discussion we had.